cpp/src/lib/stuff.cc

```diff
@@ -18,102 +18,101 @@ limitations under the License.
 #include <exception>
 #include <iomanip>
 
-namespace alba{
-namespace stuff{
-
-uint64_t get_file_size(const std:: string& file_name){
-    struct stat stat_buf;
-    int rc = stat(file_name.c_str(), &stat_buf);
-    if (rc != 0 ) {
-        std::exception e;
-        throw e;
-    }
-    uint64_t size = stat_buf.st_size;
-    return size;
+namespace alba {
+namespace stuff {
+
+uint64_t get_file_size(const std::string &file_name) {
+  struct stat stat_buf;
+  int rc = stat(file_name.c_str(), &stat_buf);
+  if (rc != 0) {
+    std::exception e;
+    throw e;
+  }
+  uint64_t size = stat_buf.st_size;
+  return size;
 }
 
-uint8_t unhex(char c){
-    uint8_t r = (uint8_t)c;
-    if ('0' <= c && c <= '9'){
-        r -= 48;
-    } else if ('a' <= c && c <= 'f'){
-        r -= 87; // 'a' = 97
-    } else{
-        throw "unhex?";
-    }
-    return r;
+uint8_t unhex(char c) {
+  uint8_t r = (uint8_t)c;
+  if ('0' <= c && c <= '9') {
+    r -= 48;
+  } else if ('a' <= c && c <= 'f') {
+    r -= 87; // 'a' = 97
+  } else {
+    throw "unhex?";
+  }
+  return r;
 }
 
-int gcd(int a, int b){
-    int t;
-
-    if (a > b) {
-        t = b;
-        b = a;
-        a = t;
-    }
-    while (b != 0){
-        t = a % b;
-        a = b;
-        b = t;
-    }
-
-    return a;
+int gcd(int a, int b) {
```

Comment:
'gcd' and 'lcm' methods seem to be dead code
(was wondering why we have these and couldn't find any usages...)